<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task</class>
 <widget class="QWidget" name="Task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCheckBox" name="checkbox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Buy Milk</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_pic">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="editButton">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Important">
     <property name="styleSheet">
      <string notr="true">border-bottom-color: rgb(239, 41, 41);</string>
     </property>
     <property name="text">
      <string>Important</string>
     </property>
     <property name="iconSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="NotImportant">
     <property name="text">
      <string>Not Important</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
